src/SDWaveFile.cpp

```diff
@@ -256,11 +304,6 @@ void SDWaveFile::readHeader()
     return;
   }
 
-  if (header.subChunk1.size != 16 || header.subChunk1.audioFormat != 1) {
```

Comment:
I don't see the sub chunk1 size and format check in find data, is this ok?

Reply:
Yes is ok, with the file supported by with this patch, the subchunk 1 could have a different size, this force to remove the size check.
About the audio format check i should reintroduce in the findData function, because i have check and in the new file the PCM is still 1 than the check could be recycled, or we can leave in this way and support also  https://github.com/bear24rw/alsa-utils/blob/master/aplay/aplay.c#L904 the float one.

Reply:
For now, I think we should not support float. We can add later if the performance impact is not too big.

Reply:
ok then i can check this on the find data and eventually return error and stop the opening of the file i have to test it but should work